src/components/SystemScoreSection.vue

```diff
@@ -44,6 +44,15 @@ export default defineComponent({
             if (this.calculatorStore.systemScore[key].label === "None") { return "No" }
             return this.calculatorStore.systemScore[key].label
         },
+        getFilterText(key) {
+            if (key === "nist" && this.calculatorStore.allNISTOptions.toLocaleString() == this.calculatorStore.activeFiltersObj.nist.sort((a, b) => a.localeCompare(b, "en", { numeric: true })).toLocaleString()) {
```

Comment:
Following the heuristic outlined above, you can simply check if the `Set` is empty to determine if all filters are selected or not.